api/handlers.go

```diff
@@ -19,6 +20,22 @@ import (
 	"gitlab.com/NebulousLabs/errors"
 )
 
+const (
+	// userLimitCacheTTL is the TTL of the entries in the userLimitCache.
+	userLimitCacheTTL = time.Hour
+)
+
+var (
+	// userLimitCacheEntry is an in-mem cache that maps from a user's sub to
+	// the data we care about.
+	userLimitCache   = map[string]userLimitCacheEntry{}
+	userLimitCacheMu sync.Mutex
```

Comment:
Similar thing as below. You are caching to be able to faster respond to api requests so this should be in the `API`. 
It's extremely rare that you ever need to put something in the global namespace except for constants or vars which are constants but can't be set as constants for some reason.